scripts/retdec_tests_runner.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+
+"""Runs all the installed unit tests."""
+
+import sys
+import os
+import subprocess
+
+import retdec_config as config
+from retdec_utils import CmdRunner
+
+"""First argument can be verbose."""
+if len(sys.argv) > 1:
+    if sys.argv[1] in ['-v', '--verbose']:
+        verbose = True
+    else:
+        verbose = False
+
+
+def print_colored(message, color):
+    """Emits a colored version of the given message to the standard output (without
+    a new line).
+       2 string argument are needed:
+       $1 message to be colored
+       $2 color (red, green, yellow)
+
+    If the color is unknown, it emits just $1.
+    """
+
+    if color == 'red':
+        print('\033[22;31m' + message + '\033[0m')
+
+    elif color == 'green':
+        print('\033[22;32m' + message + '\033[0m')
+
+    elif color == 'yellow':
+        print('\033[01;33m' + message + '\033[0m')
+
+    else:
+        print(message + '\n')
+
+
+def unit_tests_in_dir(path):
+    """Prints paths to all unit tests in the given directory.
+    1 string argument is needed:
+        path-path to the directory with unit tests
+    """
+
+    tests = []
+
+    for file in os.listdir(path):
+        file_name = os.path.basename(file)
+        if file_name.startswith('retdec-tests-') and not file.endswith('.sh'):
```

Comment:
here you forgot about adding ".py" files
`
        if file_name.startswith('retdec-tests-') and not file.endswith('.sh') and not file.endswith('.py'):
`

Reply:
Thanks for the help however I'm not quite sure if the code I wrote is even equivalent to what the original bash code intended to do:
```find "$1" -name "retdec-tests-*" -type f $EXECUTABLE_FLAG | grep -v '\.sh$' | sort```

 Sadly there are also some other problems that prevent retdec_test_runner from working, because  I'm currently running the executables incorrectly.

Reply:
yes because you added python files to the folder we should skip them when searching for tests.